index.js

```diff
@@ -6,14 +6,13 @@ var AWS = require("aws-sdk"),
 	fs = require("fs"),
 	request = require("request"),
 	path = require("path"),
+	sharp = module.parent.require("sharp"),
```

Comment:
You're right with your thought but let me explain the reason of using the import in that way:

If I use ```sharp = require("sharp")``` it works fine in my local dev environment as it should, but when I use the same import statement in Heroku containers, the index.js of the plugin throws an error saying that sharp package could not be found.
I also saw that "sharp" package was being installed inside the "nodebb -> node_modules" folder and not in "nodebb -> node_modules -> s3-uploader -> node_modules" after a clean nodebb install.

With ```sharp = module.parent.require("sharp")``` approach, it works in both local devel and Heroku containers.

Why?
Well... it's a weird behaviour that I can't explain better, but it works